pkg/providers/cloudstack/cloudstack.go

```diff
@@ -440,9 +440,11 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, datacenterConfigSpec v1alpha1
 		"cloudstackAccount":                      datacenterConfigSpec.Account,
 		"cloudstackControlPlaneComputeOffering":  controlPlaneMachineSpec.ComputeOffering.Value,
 		"cloudstackControlPlaneTemplateOffering": controlPlaneMachineSpec.Template.Value,
+		"cloudstackControlPlaneDetails":          controlPlaneMachineSpec.UserCustomDetails,
```

Comment:
Can we add `custom` to these field names? `cloudstackControlPlaneCustomDetails`, similarly for the 2 below.